### PR TITLE
Improvised Footer and Documentation

### PR DESCRIPTION
**Pull request by jyotsnad246**
## Changes Done
image

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/good-first-issue-finder/pull/178"></a>

### CODE REVIEW

File: src/lib/components/footer.svelte [hunk 1]
```diff
@@ -17,4 +17,8 @@
       >
     </li>
   </ul>
+  <br />
```
Comment (schmelto) [inline src/lib/components/footer.svelte:20]: Best practice is to use css to style the components and not `<br>`.

File: src/lib/components/footer.svelte [hunk 2]
```diff
@@ -17,4 +17,8 @@
...
+  <div class="flex list-none items-center justify-center gap-8">
```
Comment (schmelto) [inline src/lib/components/footer.svelte:21]:
```suggestion
  <div class="flex list-none items-center justify-center">
```

not sure weather this is needed or not.

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

File: src/lib/components/footer.svelte (hunk 1 above)
Comment (jyotsnad246) [reply src/lib/components/footer.svelte:20]: Okay, I'll remember this from next time.

**Comment by Cahllagerfeld** [comment]
Hey @jyotsnad246, I agree with the comments from @schmelto. Just a small ping :) Let me know if you need help with it.

**Comment by jasham** [comment]
@schmelto is this bug is still open. Can I work on this if not fixed yet?

**Review by Cahllagerfeld — COMMENTED** [review]
We've recently released Version 2 of the project which included a revamp of the footer. 
- I think we can still add the `Made with Love....` to the footer. The other changes aren't needed anymore I think. 
- The changes to the readme look good to me 👍

**Comment by Cahllagerfeld** [comment]
Just a small ping :) Let me know if you need help with that

**Comment by Cahllagerfeld** [comment]
closing the PR due to inactivity. Feel free to reopen 👍

**Pull request closed without merging**